app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -101,6 +101,15 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
 
             val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
 
+            val trackingProtection = application.settings().run {
+                when {
+                    !shouldUseTrackingProtection -> null
```

Comment:
The default tracking protection setting is ON. Does this reflect that setting?

Reply:
This doesn't reflect the default, just the raw value. This is indended.